ArduSub/Log.cpp

```diff
@@ -178,6 +178,7 @@ struct PACKED log_GuidedTarget {
     LOG_PACKET_HEADER;
     uint64_t time_us;
     uint8_t type;
+    uint8_t alt_frame;
```

Comment:
Have you thought about making this a Location::AltFrame instead of uint8_t?

Reply:
Hmmm, will all compilers map Location::AltFrame to uint8_t? I don't see this usage anywhere else in the code.

Reply:
I've used a static cast